src/monster.h

```diff
@@ -64,17 +64,19 @@ class Monster final : public Creature
 			}
 		}
 
-		void removeList() override;
 		void addList() override;
+		void removeList() override;
+
+		const std::string& getName() const override;
+		void setName(std::string name);
+
+		const std::string& getNameDescription() const override;
+		void setNameDescription(std::string nameDescription) {
```

Comment:
Const reference for nameDescription argument?
`void setNameDescription(const std::string& nameDescription)`